bin/waagent2.0

```diff
@@ -2156,6 +2157,132 @@ class FreeBSDDistro(AbstractDistro):
     def routeAdd(self, net, mask, gateway):
         Run("/sbin/route add -net " + net + " " + mask + " " + gateway, chk_err=False)
 
+
+############################################################
+#	OpenWRTDistro
```

Comment:
We are currently not supporting waagent2.0 version of the agent. This code is deprecated and you can remove this change from the PR.

Reply:
OK, I did removed the wagent2.0 change